src/workers/services/workerCryptoService.ts

```diff
@@ -0,0 +1,57 @@
+import { EncryptionType } from '../../enums';
+import { CipherString, SymmetricCryptoKey } from '../../models/domain';
+import { NodeCryptoFunctionService } from '../../services/nodeCryptoFunction.service';
+import { WorkerLogService } from './workerLogService';
+
+export class WorkerCryptoService {
+    cryptoFunctionService = new NodeCryptoFunctionService();
```

Comment:
So we can run node APIs in a web worker?

Reply:
Yes